docs/csharp/programming-guide/concepts/async/index.md

```diff
@@ -42,6 +45,9 @@ Let's start by updating this code so that the thread doesn't block while tasks a
 
 [!code-csharp[SimpleAsyncBreakfast](./snippets/index/AsyncBreakfast-V2/Program.cs#Main)]
```

Comment:
Maybe use the `:::code .... :::` syntax to be consistent with line 27??

Reply:
Some of them didn't require a change, but it's not a bad idea to update it.